pkg/loop/plugin_mercury_test.go

```diff
@@ -9,8 +9,7 @@ import (
 
 	"github.com/smartcontractkit/chainlink-common/pkg/logger"
 	"github.com/smartcontractkit/chainlink-common/pkg/loop"
-	mercury_common_test "github.com/smartcontractkit/chainlink-common/pkg/loop/internal/mercury/common/test"
-	mercury_test "github.com/smartcontractkit/chainlink-common/pkg/loop/internal/mercury/common/test"
+	mercury_test "github.com/smartcontractkit/chainlink-common/pkg/loop/internal/relayer/providerext/mercury/test"
```

Comment:
Can we try to avoid underscores?
```suggestion
	mercurytest "github.com/smartcontractkit/chainlink-common/pkg/loop/internal/relayer/providerext/mercury/test"
```
https://go.dev/doc/effective_go#names

Reply:
i have this ticket to make another cleanup of the tests https://smartcontract-it.atlassian.net/browse/BCF-3068